### ISSUE TITLE
tinymce is not defined

### ISSUE THREAD

**Issue by alkuhlani** [shridarpatil/frappe_tinymce#2]
When I use it offline, I get this erro
"tinymce is not defined"

**Comment by shridarpatil** [comment]
Yes, We use TinyMCE CDN. If you want to use it offline, you must make some app changes.

https://github.com/shridarpatil/frappe_tinymce/blob/master/frappe_tinymce/hooks.py#L20

**Comment by alkuhlani** [comment]
Yes. I do it, I will share the changes as Pull Request. thank you in advance.
